src/styles/main.scss

```diff
@@ -1,3 +1,55 @@
+@import"variables";
+
+html {
+  font-family: "Arial", "Helvetica", sans-serif;
```

Comment:
Define the font-family in the body selector instead of the html selector for better CSS specificity.